Trivial change: Misleading zip64 limit error message

dmjohnsson23 opened the pull request:
This error message is confusing as it seems to indicate that the file is already using zip64. This new message makes it more clear what the problem is, and will help guide developers to find the solution. (That is, to use in `ZipFile.open(force_zip64=True`))

kumaraditya303 reviewed (changes requested):
There are merge conflicts.

dmjohnsson23 committed:
Change confusing error message

This error message can mistakenly confuse people into thinking that the zip file is already using zip64, when in fact the problem is that they are not using zip64.

kumaraditya303 committed:
Merge branch 'main' into patch-1
